testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py

```diff
@@ -12,33 +11,45 @@ def credentials(request):
 
 
 @pytest.fixture(scope="module")
-def authorization(rhsso, openshift, blame, route, module_label, credentials):
-    """Add RHSSO identity to AuthConfig"""
-    authorization = AuthConfig.create_instance(openshift, blame("ac"), route, labels={"testRun": module_label})
+def authorization(authorization, rhsso, credentials):
+    """Add RHSSO identity to Authorization"""
+    authorization.identity.clear_all()
     authorization.identity.add_oidc("rhsso", rhsso.well_known["issuer"], credentials=Credentials(credentials, "Token"))
     return authorization
 
 
 def test_custom_selector(client, auth, credentials):
     """Test if auth credentials are stored in right place"""
     response = client.get("/get", headers={"authorization": "Token " + auth.token.access_token})
-    assert response.status_code == 200 if credentials == "authorization_header" else 401
```

Comment:
Wouldn't it be sufficient to just put brackets like this? 
`assert response.status_code == (200 if credentials == "authorization_header" else 401)`

Reply:
It would, I think it is more readable with separate if